Libplanet/Net/Messages/Message.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Libplanet.Crypto;
+using NetMQ;
+
+namespace Libplanet.Net.Messages
+{
+    internal abstract class Message
+    {
+        protected enum MessageType : byte
+        {
+            /// <summary>
+            /// Check message to determine peer is alive.
+            /// </summary>
+            Ping = 0x01,
+
+            /// <summary>
+            /// Reply of `Ping`.
+            /// </summary>
+            Pong = 0x02,
+
+            /// <summary>
+            /// Peer's delta set to sync peer list.
+            /// </summary>
+            PeerSetDelta = 0x03,
+
+            /// <summary>
+            /// Request to query block hashes.
+            /// </summary>
+            GetBlocks = 0x4,
```

Comment:
Though it's really trivial, values for `Ping`/`Pong`/`PeerSetDelta` follow `0x0?` style (note that zero), while `GetBlocks`/`Inventory` follow `0x?` style.  It could follow the consistent style.